### ISSUE TITLE
[Enhancement] Modify Velocity When launch

### ISSUE THREAD

**Issue by ronyan149** [promcteam/proskillapi#532]
Describe the solution you'd like
Add Mechanic: Velocity
Can increase or decrease velocity when use bow or launch an item? Like enchant rapid of plugins Enchantmentsapi. As long as when is launch a projectile from the player, the velocity of the projectile can be adjusted.

**Comment by Sentropic** [comment]
I don't think this makes sense as a mechanic, mechanics are something that occur at a specific time, like launching something, not increasing a number for when something else happens.

Perhaps this would make more sense as a stat you can tweak with attributes? Or as a https://github.com/promcteam/prorpgitems stat

**Comment by ronyan149** [comment]
Yeah, whatever do you want! whenever player launch a projectile, player can use something to modify velocity of that projectile.

**Comment by fructozer** [comment]
how about adding 'postProcess' to 'LaunchTrigger'. Use the 'api-velocity' value to readjust the velocity of the projectile.

**Comment by ronyan149** [comment]
> how about adding 'postProcess' to 'LaunchTrigger'. Use the 'api-velocity' value to readjust the velocity of the projectile.

I think if it's an attribute it will be easy for players to use
